valispace/__init__.py

```diff
@@ -288,10 +463,11 @@ def update_matrix_formulas(self, id, matrix):
 
 	# Private methods
 	def __name_to_id(self, name):
-		valis = self.all_vali_names()
-		for vali in valis:
-			if vali["name"] == name:
-				return vali["id"]
+		url = self.valispace_login['url'] + "vali/?name=" + str(name)
```

Comment:
Shouldn't this be ?shortname=

Reply:
@simonvbu about this: if get_vali is supposed to return a single vali (with id or name), then it should be "name" right? Or maybe you are saying that we should also have a "name" filter? :)

Reply:
That's right, it's fine with name for the single vali, for the filter it would be shortname (would indeed be nice to have).